server/historian/README.md

```diff
@@ -21,6 +21,24 @@ develop outside of the container.
 `npm install`
 `npm run build`
 
+Alternatively, for development, start a mounted container using
+
+```shell
+npm run start:dev
+```
+
+Then, when making changes, to update the running container code use
+
+```shell
+npm run build
+docker-compose restart historian
+```
+
+To run in combination with Routerlicious for easy end-to-end testing
+1. Comment out all services in `docker-compose.yml` except for historian, then save.
```

Comment:
It's the other way around right? Comment out historian and keep everything else?